style.css

```diff
@@ -1 +1,158 @@
 /*  Add your styling here */
+* body {
+  font-family: "Mochiy Pop One", sans-serif;
+  display: grid;
```

Comment:
Have a little look at how to use CSS grid. One way of doing it is to set up a wrapper in your HTML:

```
<div class="wrapper">
    <div class="header">Header</div>
    <div class="sidebar">Sidebar</div>
    <div class="content">Content</div>
    <div class="footer">Footer</div>
</div>
```

Then make sure each of those classes has a grid-name in the CSS:

```
.header {
    grid-area: hd;
}
.footer {
    grid-area: ft;
}
.content {
    grid-area: main;
}
.sidebar {
    grid-area: sd;
}
```

Then finally, define the layout on the grid wrapper (the parent class above):
```
.wrapper {
    display: grid;
    grid-template-columns: repeat(9, 1fr); <= this is giving you 9 columns which are of equal width
    grid-auto-rows: minmax(100px, auto); <= this is giving the rows a minimum height of 100px and a maximum height that just scales to whatever is in it
    grid-template-areas:
      "hd hd hd hd   hd   hd   hd   hd   hd"
      "sd sd sd main main main main main main"
      "ft ft ft ft   ft   ft   ft   ft   ft";
}
```